alembic/env.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import with_statement
```

Comment:
From what I'm seeing in the FSF's guidelines, I think we should put a basic copy right line (year, 'massachusetts open cloud contributors'), and then a notice that the code was largely autogenerated, to make it clear.  The FSF doesn't list any exception to license-headers for boilerplate files; only one for files under 10 lines.